CMS-Enterprise/easi-app
# Easi 4580/email notifications

# EASI-4850

## Description

- add Yes/No selection for admins to choose to send notifications when uploading docs


## How to test this change

- nav to admin view of Supporting Documents on a system intake
- upload a doc
- confirm you see the below
- NOTE: the below will _never_ show when initially submitting the system intake as the user creating the intake will _always_ have the `requester` role (even if the user is an admin)
<img width="983" alt="Screenshot 2024-08-21 at 12 31 20" src="https://github.com/user-attachments/assets/fd364684-f377-402c-b80f-2119717b0ebb">

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
